Bukkit/src/main/java/net/dries007/mclink/MCLinkAuthEvent.java

```diff
@@ -0,0 +1,38 @@
+package net.dries007.mclink;
+
+/*
+ * Copyright (c) 2018 Dries007 & Brian Ronald. All rights reserved
```

Comment:
Copyright assignment: I'm explicitly giving Dries007 full joint copyright on this file, so that it can be included in its entirety within MCLink without fear of future licensing problems, even from me.